treesit-auto.el

```diff
@@ -240,6 +240,11 @@ automatic installation (or prompting, based on the value of
       :ts-mode 'toml-ts-mode
       :remap '(conf-toml-mode toml-mode)
       :url "https://github.com/tree-sitter/tree-sitter-toml")
+    ,(make-treesit-auto-recipe
+      :lang 'dart
```

Comment:
I realize it's a bit of a nitpick, but the other recipes are all in alphabetical order so that it's easy to visually assess what's tracked in the file.  Mind just moving this recipe up to between `css` and `dockerfile`?  Thanks for contributing!

Reply:
updated